docs/explanation/upgrade.rst

```diff
@@ -0,0 +1,13 @@
+=======
+Upgrade
+=======
+
+This phase is responsible for applying the upgrade plan step by step (See
+:doc:`Upgrade a cloud <../how-to/upgrade-cloud>`). By default steps are applied
+sequentially, but there are steps which can be run in parallel.
+
+Required Juju commands
```

Comment:
what are we trying to communicate here? "Required" makes it sound like the user has to do something about these commands

Reply:
The idea was to refer to permission level for Juju user. However, this maybe be mention only in `Get started` section. What do you say?

Reply:
ah I understand. Then I think we should be either more specific or remove this part. Given the current permission model, I imagine that the juju user needs to have `write` permission on the model containing the OpenStack deployment